airflow/operators/python.py

```diff
@@ -276,6 +276,16 @@ def task(
     :type multiple_outputs: bool
 
     """
+
+    if python_callable:
+        sig = signature(python_callable).return_annotation
+        ttype = getattr(sig, "__origin__", None)
+        if (
+            sig != inspect.Signature.empty
+            and (ttype in (dict, Dict, tuple, Tuple))
```

Comment:
```suggestion
            and ttype in (dict, Dict, tuple, Tuple))
```
I would say that we should only check for `(tuple, Tuple)` because that's the only indicator of multiple output. For example:
```
def x() -> Dict[str, str]:
    return {"a": "b", "c": "d'}
```
is a single output function in my opinion.
WDYT? @casassg @evgenyshulman @jonathanshir